src/s2/util/math/exactfloat/exactfloat.cc

```diff
@@ -125,6 +127,30 @@ static int BN_ext_count_low_zero_bits(const BIGNUM* bn) {
   return count;
 }
 
+#else
+
+static int BN_ext_count_low_zero_bits(const BIGNUM* bn) {
+  // BN_bn2lebinpad(const BIGNUM *a, unsigned char *to, int tolen)
```

Comment:
I don't think you need to copy the BN_bn2lebinpad spec, people can look it up if they're wondering. If you think there's something particularly remarkable or surprising, comment that.